Add babel-runtime to dependencies

I tried to create a new nodejs Electron app with Gridicons by adding the package using npm. When run, I get this error: `Cannot find module 'babel-runtime/core-js/object/get-prototype-of'`. 

It is, indeed, at the very beginning of the compiled Gridicons `build/index.js` file after building:

```
var _getPrototypeOf=require('babel-runtime/core-js/object/get-prototype-of')
```

Installing the `babel-runtime` package from npm solves the issue.

I'm not sure why it's required, but perhaps it should be a dependency or a peer dependency?

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -209,7 +209,6 @@ module.exports = function( grunt ) {
 				],
 				comments: false,
 				plugins: [
-					'transform-runtime',
 					'transform-class-properties',
 					'transform-export-extensions',
 					'add-module-exports',

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "gridicons",
-  "version": "1.0.1-alpha.1",
+  "version": "1.1.0-alpha.1",
   "main": "build/index.js",
   "scripts": {
     "build": "grunt --verbose",
@@ -25,7 +25,6 @@
     "babel-plugin-transform-export-extensions": "^6.8.0",
     "babel-plugin-transform-react-display-name": "^6.8.0",
     "babel-plugin-transform-react-jsx": "^6.8.0",
-    "babel-plugin-transform-runtime": "^6.15.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-stage-2": "^6.18.0",
     "babili": "0.0.10",

